vcs/vcsvtk/pipeline1d.py

```diff
@@ -103,6 +103,7 @@ def plot(self, data1, data2, tmpl, grid, transform, **kargs):
                 self._context().canvas.plot(m, donotstoredisplay=True)
 
         ren2 = self._context().createRenderer()
+        ren2.SetLayer(1)
```

Comment:
@danlipsa thinking back on this one. Shouldn't the layer be determined either by `template.data.priority` or `object.priority`